README.md

```diff
@@ -0,0 +1,300 @@
+
+### Why Open Source?
+
+Why is Monica open source? Is it risky? Will someone steal my code and do a for-profit business that will kill my own business? Why reveal my strategy to the world? These are the kind of questions we’ve received by email already.
+
+The answer to these questions is simple: yes, you can fork Monica and make a competing project, make money out of it (even if the license is not super friendly towards that) and I’ll never know. But it’s okay, I don’t mind.
+
+I wanted to open source Monica for several reasons:
+
+- **I believe that this tool can really change people’s lives.**  
+    While I aim to make money out of it, I also want everyone to benefit from it. Open sourcing a project like this will help Monica become much bigger than what I imagine myself. While I strongly believe that this software has to follow the vision I have for it, I need to be humble enough to know that ideas come from everywhere, and people have much better ideas than what I can have.
+
+- **You can’t make something great alone.**  
+    While Monica could become a company and hire a bunch of super smart people to work on it, you can’t beat the manpower of an entire community. Open sourcing the product means bugs will be fixed faster, features will be developed faster, and more importantly, developers will be able to contribute to a tool that positively changes their own lives and the lives of other people.
+
+- **Doing things in a transparent way leads to formidable things.**  
+    People respect the project more when they can see how it’s being worked on. You can’t hide nasty things in the code. You can’t do things behind the backs of your users. Doing everything in the open is a major driving force that motivates you to keep doing what’s right.
+
+- **Once you’ve created a community of passionate developers around your project, you’ve won.**  
+    Because developers are very powerful influencers. Developers will create apps around your product, talk about it on forums, and share the project with their friends, families, and colleagues. Cherish the developers – users will follow.
+
+### Patreon
+
+You can support the development of Monica [on Patreon](https://www.patreon.com/monicahq). Thanks for your help.
+
+## Contact
+
+## Team
+
+Our team is made of three core members:
+
+- [Regis Freyd (djaiss)](https://github.com/djaiss)
+
+- [Théo Matthieu (mokto)](https://github.com/mokto)
+
+- [Alexis Saettler (asbiin)](https://github.com/asbiin)
+
+We are also fortunate to have an amazing [community of developers](https://github.com/monicahq/monica/graphs/contributors) who help us greatly.
+
+## Thank you, open source
+
+Monica uses a lot of open source projects and we thank them with all our hearts. We hope that providing Monica as an free, open source project will help other people the same way those softwares have helped us.
+
+## License
+
+Copyright (c) 2016–2020
```

Comment:
We should probably increase the year (yeah)
```suggestion
Copyright (c) 2016–2021
```

Reply:
Good spot. Fixed.